### PR TITLE
Aesthetics fixups

### PR DESCRIPTION
**Pull request by 10000TB**
Signed-off-by: David Hu <xuehaohu@google.com>

### CODE REVIEW

**Comment by 10000TB** [comment]
@yurinnick -- it would be helpful to have you fact check me on the comments added for Ziron image parsing. Those are just for the pkg to be compliant with go styles, or our internal presubmit style checks are going crazy

**Review by hugelgupf — APPROVED** [review]

File: pkg/boot/zbi/zbi.go [hunk 1]
```diff
@@ -100,21 +142,25 @@ var (
...
 
+// ToString return string representation of current ZBIType.
 func (it *ZBIType) ToString() (string, error) {
 	if typeMetadata, ok := ZBITypes[*it]; ok {
 		return typeMetadata.Name, nil
 	}
 	return "", fmt.Errorf("Can't find metadata for %#08x ZBIType", it)
 }
 
+// MarshalJSON returns JONS bytes of current ZBIType.
```
Comment (yurinnick) [inline pkg/boot/zbi/zbi.go:163]: `JSON`

File: pkg/boot/zbi/zbi.go [hunk 2]
```diff
@@ -191,11 +241,13 @@ func (i *Image) readContainerHeader(f io.ReadSeeker) error {
 	return nil
 }
 
+// ZBIKernel abstracts an in-memory kernel entry.
 type ZBIKernel struct {
 	Entry             uint64
 	ReserveMemorySize uint64
 }
 
+// ZirconKernel abstracts a Zircon kernel file.
```
Comment (yurinnick) [inline pkg/boot/zbi/zbi.go:250]: If I understand [image.h](https://cs.opensource.google/fuchsia/fuchsia/+/main:zircon/system/public/zircon/boot/image.h) correctly `ZirconKernel` abstracts kernel loaded into memory, while `ZBIKernel` specifies Zircons kernel entry point and entrypoint.

Comment (yurinnick) [reply pkg/boot/zbi/zbi.go:250]: In [boot-zbi.h](https://cs.opensource.google/fuchsia/fuchsia/+/main:zircon/kernel/phys/include/phys/boot-zbi.h;l=102) `ZBIKernel` addressed as kernel header and `ZirconKernel` as kernel image. I find the this naming situation quite confusing.

File: pkg/boot/zbi/zbi.go (hunk 1 above)
Comment (10000TB) [reply pkg/boot/zbi/zbi.go:163]: Done.

File: pkg/boot/zbi/zbi.go (hunk 2 above)
Comment (10000TB) [reply pkg/boot/zbi/zbi.go:250]: I'll just copy its current comment then, `// The whole contiguous image loaded into memory by the boot loader.`

**Review by hugelgupf — APPROVED** [review]

**Pull request merged**